pkgs/servers/tailscale/default.nix

```diff
@@ -1,4 +1,15 @@
-{ lib, stdenv, buildGoModule, fetchFromGitHub, makeWrapper, iptables, iproute2, procps, shadow, getent }:
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+, getent
+, iproute2
+, iptables
+, shadow
+, procps
+, nixosTests
+}:
 
 let
   version = "1.58.0";
```

Comment:
We would need a rebase since 1.58.2 is merged:
- https://github.com/NixOS/nixpkgs/pull/283357